applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -45,12 +45,14 @@ def AddVariables(self):
         super(ExplicitMechanicalSolver, self).AddVariables()
         self._add_dynamic_variables()
         self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.MIDDLE_VELOCITY)
+        self.main_model_part.AddNodalSolutionStepVariable(StructuralMechanicsApplication.FRACTIONAL_ACCELERATION)
```

Comment:
1. Is this needed to be a historical variable?
2. If yes then please add it only if the "multi_stage" scheme is used